lib/recipient.rb

```diff
@@ -0,0 +1,30 @@
+class Recipient
+  attr_reader :slack_id, :name
+  
+  def initialize(slack_id: , name: )
+    @slack_id = slack_id
+    @name = name
+  end
+  
+  def self.get(url, params)
+    query = {
+      token: params 
+    }
+  
+    response = HTTParty.get(url, query: query)
+    
+    if response["error"]
```

Comment:
You should also check `response.code` here.

```suggestion
    if response["error"] || response.code != 200
```